IDEAS/Fluid/HeatExchangers/RadiantSlab/EmbeddedPipe.mo

```diff
@@ -189,16 +189,15 @@ initial equation
   end if;
 
 equation
-  // simplify expression for sufficiently large mass flow rates
-  if useSimplifiedRt then
-    // Koschenz eq 4-60
-    R_t = (IDEAS.Utilities.Math.Functions.inverseXRegularized(2*m_flowSpLimit*cp_default*nDiscr, deltaXR) + R_w_val + R_r_val + R_x_val);
-  else
-    // Koschenz eq 4-59
-    R_t = (IDEAS.Utilities.Math.Functions.inverseXRegularized(m_flowSpLimit*cp_default*nDiscr*(1-exp(-1/((R_w_val+R_r_val+R_x_val+R_c)*m_flowSpLimit*cp_default*nDiscr))), deltaXR)-R_c);
-  end if;
+  // this need not be smooth since when active, G_max is already active
+  m_flowSpLimit = max(m_flowSp, 1e-8);
```

Comment:
Since that max function will/should never be differentiated, since there is another regularisation that 'overwrites' it.